modules/conversantBidAdapter.js

```diff
@@ -225,17 +225,11 @@ export const spec = {
 };
 
 function getPubcid(bidRequest) {
-  let pubcid = null;
-  if (bidRequest.userId && bidRequest.userId.pubcid) {
-    pubcid = bidRequest.userId.pubcid;
-  } else if (bidRequest.crumbs && bidRequest.crumbs.pubcid) {
-    pubcid = bidRequest.crumbs.pubcid;
-  }
-  if (!pubcid) {
-    const pubcidName = getBidIdParameter('pubcid_name', bidRequest.params) || '_pubcid';
-    pubcid = readStoredValue(pubcidName);
-  }
-  return pubcid;
+  const pubcid = Number(bidRequest.userIdAsEids?.pubcid?.[0].uids?.[0].id);
```

Comment:
why convert to a number? that seems dangerous

Reply:
I think you will generally get NaN as the result? It isnt usually a number

Reply:
After talking it over here we're going to look for the userid from the eids, and not use our own user.ext.fpc.  This also means we have no need to directly read cookies either so that's been removed as well